### PR TITLE
New version: Postman 9.31

### PR DESCRIPTION
**Pull request by sootysec**
Signed-off-by: Daniel Sutton <daniel@sootysec.uk>

- [x ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/83228)

### CODE REVIEW

File: manifests/p/Postman/Postman/9.31/Postman.Postman.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Postman.Postman
+PackageVersion: 9.31
```
Comment (mdanish-kh) [inline manifests/p/Postman/Postman/9.31/Postman.Postman.installer.yaml:5]:
```suggestion
PackageVersion: 9.31.0
```
This should match Control Panel.

File: manifests/p/Postman/Postman/9.31/Postman.Postman.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Postman.Postman
+PackageVersion: 9.31
```
Comment (mdanish-kh) [inline manifests/p/Postman/Postman/9.31/Postman.Postman.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 9.31.0
```
This should match Control Panel.

File: manifests/p/Postman/Postman/9.31/Postman.Postman.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Postman.Postman
+PackageVersion: 9.31
```
Comment (mdanish-kh) [inline manifests/p/Postman/Postman/9.31/Postman.Postman.yaml:5]:
```suggestion
PackageVersion: 9.31.0
```
This should match Control Panel.

**Comment by mdanish-kh** [comment]
You would also have to change the folder name `9.31` -> `9.31.0`. You can do this here in GitHub by pressing `.` (dot) on your keyboard that'll open up VSCode Online.

**Comment by sootysec** [comment]
@mdanish-kh Thank you

**Comment by ItzLevvie** [comment]
Close with reason: Replaced by PR 84632;

**Pull request closed without merging**